lib/MetaCPAN/Query/File.pm

```diff
@@ -63,7 +63,6 @@ sub interesting_files {
                                 {
                                     bool => {
                                         must => [
-                                            { term => { level => 0 } },
```

Comment:
Regarding this change here, can you remind me why we need to do this?  Will these new files not generally be at the top level?

Reply:
Yep in https://github.com/metacpan/metacpan-api/pull/803/commits/78bfcb6d2fc2280683f1df75452fb2dc446ff967

I wanted to match places like https://metacpan.org/pod/distribution/Moose/lib/Moose/Manual/Contributing.pod as well, hence this change.

Reply:
Ah, that makes sense to me.  @mickeyn does that work for you?  I don't know of any performance implications, but maybe you do?

Reply:
Yeah, let me know if there's a better/more maintainable way :muscle:

Reply:
Given the filenames matched, this seems like it should be ok, but I'm still wary. Will the pod file in question be displayed as 'Moose/Manual/Contributing.pod' for example if a distribution has several such files throughout the hierarchy?

Reply:
Yes, Moose::Manual::Contributing still shows up under Documentation

![image](https://user-images.githubusercontent.com/110625/39852182-aa171fca-544d-11e8-8dfc-27c55c8079e2.png)

Reply:
I think that removing this from the query has caused the problems I was worried about, where it picks up various internal copies that are not as interesting as the one at root level. Example: https://metacpan.org/release/Dist-Zilla#other

Reply:
Maybe we just want to filter the results to prefer the file which is closest to the root of the directory?

Reply:
@Grinnz @oalders thanks, I'll look into this and try to get an PR out by the weekend :muscle: